src/Common/src/Internal/Cryptography/OpenSslAsymmetricAlgorithmCore.cs

```diff
@@ -72,7 +71,75 @@ private static HashAlgorithm GetHashAlgorithm(HashAlgorithmName hashAlgorithmNam
 
             return hasher;
         }
-    }
-}
 
+        public static byte[] ConvertToOpenSslFormat(byte[] input)
```

Comment:
The Windows form is, apparently, from IEEE 1363.  So this might be ConvertIeee1363ToDer (and the other function named similarly).  And even then I wish I could find a copy of the document to find out if it's just a DSA format, or what.  (Mainly I'm concerned with an overly-general name here, given that this is just for DSA and ECDSA)